### ISSUE TITLE
ENS Notification Banner

### ISSUE THREAD

**Issue by brlodi** [Pitt-CSC/club_project_2017#112]
Not that this thought was triggered by anything in particular (boil your water people! *Really*. The PA DEP doesn't issue boil water advisories unless they really mean it), but it might be cool to have recent Emergency Notification System messages displayed banner-style at the top of the map. They're a very different kind of data from our mapped stuff, but arguably more important.

We'd just have to figure out how to get it, ideally without tying it to any club member's email or contact phone number.

**Comment by vonbearshark** [comment]
Yeah similar idea here: #27 . I think this is a great feature idea to at least look in to

**Comment by brlodi** [comment]
@vonbearshark Whoops didn't see that issue when I searched
